refactor: typo (#535)
oceanroleplay/discord.ts: packages/discordx/README.md

# 💡 Why discordx?

> With `discordx`, we intend to provide latest up to date package to build bots with many features, such as multi-bot, simple commands, pagination, music etc. **Updated daily with discord.js changes**.

Reviewer: Should be "the  latest up to date", it's missing "the"?

Reviewer 2: ```suggestion
With `discordx`, we intend to provide the latest up-to-date package to easily build feature-rich bots with multi-bot compatibility, simple commands, pagination, music, and much more. **Updated daily with discord.js changes**.
```

```ts
@Discord()
class commandTest {

>   @SimpleCommand("perm_check", { aliases: ["p_test"] })

Author: ```suggestion
  @SimpleCommand("perm-check", { aliases: ["p-test"] })
```
